packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -47,8 +47,9 @@ export interface TerminalWidgetFactoryOptions extends Partial<TerminalWidgetOpti
 export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget {
 
     private readonly TERMINAL = 'Terminal';
+    protected terminalKind = 'user';
+    protected termId = -1;
```

Comment:
please don't shorten but rename it to `_terminalId`, see https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#underscore_accessors